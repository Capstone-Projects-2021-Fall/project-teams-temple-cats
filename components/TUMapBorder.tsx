import * as React from 'react';
import { Polygon } from 'react-native-maps';

export default function TUMapBorder() {
  return (
    <Polygon
      coordinates={[
        { latitude: 39.975237221562914, longitude: -75.16531142948794 },
        { latitude: 39.99028527887604, longitude: -75.16201582672468 },
        { latitude: 39.98821021677819, longitude: -75.14598521349043 },
        { latitude: 39.9731936067945, longitude: -75.14928693177747 },
      ]}
      strokeWidth={2}
      strokeColor="rgba(157, 34, 53, 1)"
      fillColor="rgba(157, 34, 53, 0.05)"
    />
  );
}
