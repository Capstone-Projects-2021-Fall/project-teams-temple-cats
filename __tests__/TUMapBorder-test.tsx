import React from 'react';
import renderer, { act } from 'react-test-renderer';

import TUMapBorder from '../components/TUMapBorder';

//Error is logging after snapshot test is finished to test coordinates, not sure why

// act(() => {

// it('Correct map border coordinates are rendered', () => {
//   const tree = renderer.create(<TUMapBorder />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
// });

// Unsure how to test map border color 