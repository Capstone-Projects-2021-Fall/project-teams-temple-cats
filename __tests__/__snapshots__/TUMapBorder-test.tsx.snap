// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correct map border cnhoordinates are rendered 1`] = `
<AIRMapPolygon
  coordinates={
    Array [
      Object {
        "latitude": 39.975237221562914,
        "longitude": -75.16531142948794,
      },
      Object {
        "latitude": 39.99028527887604,
        "longitude": -75.16201582672468,
      },
      Object {
        "latitude": 39.98821021677819,
        "longitude": -75.14598521349043,
      },
      Object {
        "latitude": 39.9731936067945,
        "longitude": -75.14928693177747,
      },
    ]
  }
  strokeColor="#000"
  strokeWidth={2}
/>
`;
